examples/flows/chat/chat-with-pdf/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "subscription_id": "d128f140-94e6-4175-87a7-954b9d27db16",
```

Comment:
may add config.json to gitignore file in repo root directory

Reply:
done.